LISA/Executor.cpp

```diff
@@ -198,12 +198,23 @@ uint32_t Executor::GetStorageDetails(const std::string& type,
             persistentUsedKB += fs::getDirectorySpace(i);
             details.persistentPath = fs::getAppsStorageDir() + i;
         }
-        details.appUsedKB = std::to_string(appUsedKB);
-        details.persistentUsedKB = std::to_string(persistentUsedKB);
+        details.appUsedKB = std::to_string(appUsedKB / 1024);
+        details.persistentUsedKB = std::to_string(persistentUsedKB / 1024);
     }
     return ERROR_NONE;
 }
 
+uint32_t Executor::GetAppDetailsList(const std::string& type,
+                          const std::string& id,
+                          const std::string& version,
+                          const std::string& appName,
+                          const std::string& category,
+                          std::vector<DataStorage::AppDetails>& appsDetailsList) const
+{
+    appsDetailsList = dataBase->GetAppDetailsList(type, id, version, appName, category);
```

Comment:
add try {} catch for exceptions that might be thrown and return error, you can do the same for GetStorageDetails

Reply:
done